lib/states/wi/madison.R

```diff
@@ -8,86 +8,137 @@ source("common.R")
 # each year, this seems reasonable.
 load_raw <- function(raw_data_dir, n_max) {
   # NOTE: "IBM" is the officers department ID
-  cit <- load_single_file(
+  cit_old <- load_single_file(
     raw_data_dir,
     "mpd_traffic_stop_request_citations.csv",
     n_max
   )
-  warn <- load_single_file(
+  warn_old <- load_single_file(
     raw_data_dir,
     "mpd_traffic_stop_request_warnings.csv",
     n_max
   )
+  
+  cit_new <- load_single_file(
+    raw_data_dir,
+    "elcis_with_demographics.csv",
+    n_max
+  )
+  
+  warn_new <- load_single_file(
+    raw_data_dir,
+    "warnings_with_demographics.csv",
+    n_max
+  )
+  
+  #new_d <- bind_rows(
+  #  cit_new$data,
+  #  warn_new$data
+  #)
+  
+  ## NOTE: The new data starts in Jan. 2018 
+  ## and the old data covers through Sept. 2017, 
+  ## so we're still missing Oct.-Dec. 2017
```

Comment:
@KenielYao Did this get taken care of at some point? If not, we can just merge as-is and add a note to the readme.

Reply:
We have new data for citations but not warnings. That can be difficult to describe (and will require additional coercion of variable names) so I've chosen to not include it for now.